jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -251,28 +251,28 @@ ignition_collection_jobs =
         'ignition_plugin-ci-ign-plugin1-windows7-amd64',
         'ignition_plugin-install-pkg-bionic-amd64',
         'ignition_plugin1-install_bottle-homebrew-amd64',
-        'ignition_rendering-ci-ign-rendering4-bionic-amd64',
-        'ignition_rendering-ci-ign-rendering4-homebrew-amd64',
-        'ignition_rendering4-install-pkg-bionic-amd64',
-        'ignition_rendering4-install_bottle-homebrew-amd64',
-        'ignition_sensors-ci-ign-sensors4-bionic-amd64',
-        'ignition_sensors-ci-ign-sensors4-homebrew-amd64',
-        'ignition_sensors4-install-pkg-bionic-amd64',
-        'ignition_sensors4-install_bottle-homebrew-amd64',
+        'ignition_rendering-ci-ign-rendering5-bionic-amd64',
+        'ignition_rendering-ci-ign-rendering5-homebrew-amd64',
+        'ignition_rendering5-install-pkg-bionic-amd64',
+        'ignition_rendering5-install_bottle-homebrew-amd64',
+        'ignition_sensors-ci-ign-sensors5-bionic-amd64',
+        'ignition_sensors-ci-ign-sensors5-homebrew-amd64',
+        'ignition_sensors5-install-pkg-bionic-amd64',
+        'ignition_sensors5-install_bottle-homebrew-amd64',
         'ignition_tools-ci-ign-tools1-bionic-amd64',
         'ignition_tools-ci-ign-tools1-homebrew-amd64',
         'ignition_tools-ci-ign-tools1-windows7-amd64',
         'ignition_tools-install-pkg-bionic-amd64',
         'ignition_tools1-install_bottle-homebrew-amd64',
-        'ignition_transport-ci-ign-transport9-bionic-amd64',
-        'ignition_transport-ci-ign-transport9-homebrew-amd64',
-        'ignition_transport-ci-ign-transport9-windows7-amd64',
-        'ignition_transport9-install-pkg-bionic-amd64',
-        'ignition_transport9-install_bottle-homebrew-amd64',
-        'sdformat-ci-sdformat10-bionic-amd64',
-        'sdformat-ci-sdformat10-homebrew-amd64',
-        'sdformat-ci-sdformat10-windows7-amd64',
-        'sdformat-install-sdformat10_pkg-bionic-amd64'
+        'ignition_transport-ci-ign-transport10-bionic-amd64',
+        'ignition_transport-ci-ign-transport10-homebrew-amd64',
+        'ignition_transport-ci-ign-transport10-windows7-amd64',
+        'ignition_transport10-install-pkg-bionic-amd64',
+        'ignition_transport10-install_bottle-homebrew-amd64',
+        'sdformat-ci-sdformat11-bionic-amd64',
+        'sdformat-ci-sdformat11-homebrew-amd64',
+        'sdformat-ci-sdformat11-windows7-amd64',
+        'sdformat-install-sdformat11_pkg-bionic-amd64'
```

Comment:
most of these jobs don't exist yet because we haven't created release branches. I tested this branch with the following dsl job, and the resulting view is missing a lot of jobs:

* https://build.osrfoundation.org/job/_dsl_ignition_collection/306/
* https://build.osrfoundation.org/view/ign-edifice/

I'll update it to what I think will work

Reply:
> most of these jobs don't exist yet because we haven't created release branches. I tested this branch with the following dsl job, and the resulting view is missing a lot of jobs:
> 
> * https://build.osrfoundation.org/job/_dsl_ignition_collection/306/
> * https://build.osrfoundation.org/view/ign-edifice/
> 
> I'll update it to what I think will work

d0c130e